draft-dns-zone-digest.xml

```diff
@@ -515,14 +516,20 @@ RR(i) = owner | type | class | TTL | RDATA length | RDATA
             The zone digest is calculated using the algorithm
             described in <xref target="calculating-calculating"/>.
             Note in particular that the Digest field of any &RRNAME;
-            RRs MUST be zeroed prior to calculation of the digest.
+            RRs MUST be zeroed prior to calculation of the digest and
+            the RRSIGs for it must be removed.
           </t>
           <t>
             The calculated digest is compared to the received digest.
             If the two digest values match, verification is considered
             successful.  Otherwise, verification MUST NOT be
             considered successful.
           </t>
+          <t>
+	    If the zone is to be served and transferred, the original
+	    Digest value of the &RRNAME; must be replace the zeroed field so
+	    downstream clients can retrieve and verify it also.
+          </t>
```

Comment:
Yeah, I see what you're getting at here.  I think this point really belongs in the (6th) item where it talks about zeroing the digest field.  That item could be worded better.